File: ios/AudioHelper.m
#import <React/RCTBridgeModule.h>

// bridging with swift
@interface RCT_EXTERN_MODULE(AudioHelper, NSObject)\
    RCT_EXTERN_METHOD(callbackMethod:(RCTResponseSenderBlock)callback)
    RCT_EXTERN_METHOD(simpleMethod:(NSString *)message)
@end

File: ios/AudioHelper.swift
//
//  AudioHelper.swift
//  AudioHelper
//
//  Created by Imron Imam Wahyudi Gunawan on 04/02/20.
//  Copyright © 2020 Facebook. All rights reserved.
//

import Foundation

@objc(AudioHelper)
class AudioHelper: NSObject {
    
    @objc(callbackMethod:)
    func callbackMethod(callback: RCTResponseSenderBlock) -> Void {
      let resultsDict = [
        "success" : true
      ];
       
      callback([NSNull() ,resultsDict])
    }
    
    @objc(simpleMethod:)
    func simpleMethod(message: String) {
      print("\(message)")
    }
    
}
